Signum.React/Scripts/Operations/CellOperation.tsx

```diff
@@ -38,6 +38,9 @@ export function CellOperationButton({ icoc: icocOrNull, onOperationClick, canExe
 
   const icoc = icocOrNull;
 
+  if (!icoc.settings?.isVisible?.(icoc) == false)
+    return null;
+

```

Comment:
If there is not isVisible defined in CellOperstionSettings but it is defined in EntityOperationSetting it should maybe be hidden by default. Same for onClick. 

Check ContextualOperationSettings.

Since this information is not known in the server side maybe render an error message

Reply:
visibility logic you mentioned added to isVisibleInCell, as well as EntityOperationSettings.cell setting